src/styles/components/_our-goal-box.scss

```diff
@@ -0,0 +1,48 @@
+@import '../abstracts/_abstracts-dir';
+
+.our-goal-box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: solid 1px $silver-chalice;
+
+  &__title {
+    display: none;
+
+    @include respond-above('md') {
+      display: block;
+      padding: rem(32px) 0 rem(16px);
+      margin: 0;
+
+      font-size: rem(28px);
+      font-weight: bold;
+      line-height: 1.22;
+    }
+
+    @include respond-above('lg') {
+      padding: rem(44px) 0 rem(24px);
+      font-size: rem(36px);
+    }
+  }
+
+  &__description {
+    box-sizing: border-box;
+    padding: rem(16px 28px);
+    margin: auto;
+
+    font-size: rem(14px);
+    font-weight: 200;
+    line-height: 1.53;
+    text-align: center;
+
+    @include respond-above('md') {
+      height: auto;
+      padding: 0 rem(28px 32px 28px);
+      font-size: rem(16px);
+    }
+
+    @include respond-above('lg') {
+      padding: 0 rem(110px 44px 110px);
```

Comment:
This could also be written like this `padding: 0 rem(110px 44px) => (top left-right bottom)`